docs/_posts/Meryem1425/2023-05-31-sbiobertresolve_icd10cm_slim_billable_hcc_en.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: model
+title: Sentence Entity Resolver for Billable ICD10-CM HCC Codes (sbiobertresolve_icd10cm_slim_billable_hcc)
+author: John Snow Labs
+name: sbiobertresolve_icd10cm_slim_billable_hcc
+date: 2023-05-31
+tags: [licensed, en, clinical, icd10cm, entity_resolution]
+task: Entity Resolution
+language: en
+edition: Healthcare NLP 4.4.2
+spark_version: 3.0
+supported: true
+annotator: SentenceEntityResolverModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps extracted clinical entities to ICD-10-CM codes using `sbiobert_base_cased_mli` sentence bert embeddings. In this model, synonyms having low cosine similarity to unnormalized terms are dropped. It returns the official resolution text within the brackets and also provides billable and HCC information of the codes in `all_k_aux_labels` parameter in the metadata. This column can be divided to get further details: `billable status || hcc status || hcc score`. For example, if `all_k_aux_labels` is like `1||1||19` which means the `billable status` is 1, `hcc status` is 1, and `hcc score` is 19.
```

Comment:
lets share Hierarchical Condition Categories (HCC) when we mention for the first time.

Reply:
Done!